.github/workflows/ecs-deploy-v2.yml

```diff
@@ -37,6 +38,12 @@ on:
       terraform_version:
         type: string
         required: false
+      use_version_as_docker_image_tag:
+        type: boolean
+        default: true
```

Comment:
Could you add the `required` block to make the behaviour explicit here, I admit I don't remember which is the default and we add it whether we set it to `false` or `true` in the other fields.